ConsentViewController/Classes/SPError.swift

```diff
@@ -69,6 +69,7 @@ import Foundation
 }
 
 @objcMembers public class InvalidArgumentError: SPError {
+    override public var spCode: String { "sp_metric_invalid_local_data" }
```

Comment:
@Nevazhnovu judging by the name of the class (`InvalidArgumentError`), are you sure that's the correct `spCode`?

Reply:
@andresilveirah 146% correct. Judging by the use case - it should be the same as in Android.
"""PropertyName can only include letters, numbers, '.', ':', '-' and '/'. \(string) 
passed is invalid"""

Reply:
Hmm, so it seems to be we need a better error code and a better error class if that's the only time that error thrown. Something in the lines of `InvalidPropertyNameError` (code: `sp_metric_invalid_property_name`).

Reply:
@andresilveirah changed to `sp_metric_invalid_property_name` but consider my previous message (it will not be unified with Android SDK this way)

Reply:
Sure, it makes sense.